src/app/app.component.ts

```diff
@@ -321,7 +354,12 @@ export class AppComponent implements AfterViewInit {
 
       try {
         const repoId = await this.repoClient.getCurrentRepoId();
-        console.log(repoId, parentEntryId, this.fileName, entryRequest)
+        const currentSelectedByPathResponse = await this.repoClient.entriesClient.getEntryByPath({
+          repoId,
+          fullPath: this.lfSelectedFolder.selectedFolderPath
+        });
+        const currentSelectedEntry = currentSelectedByPathResponse.entry;
+        const parentEntryId = this.getIdOrTargetId(currentSelectedEntry);
```

Comment:
the  getIdORTargetId was added to fix the previous bug right? Is it still needed?